scripts/create-archive

```diff
@@ -18,11 +20,24 @@ SOURCE_DATE_EPOCH=$(git log -1 --format=%ct)
 # - excluding .git, pkgs folder and prevent probable not so
 #   clever implementation of 'tar' which would result in
 #   an infinity loop due to tar '.'
-tar --sort=name \
-    --mtime="@${SOURCE_DATE_EPOCH}" \
-    --owner=0 --group=0 --numeric-owner \
-    --xform="s%^\./%${GIT_ARCHIVE_PREFIX}%" \
-    --exclude pkgs --exclude .git* --exclude "${GIT_TARBALL_NAME}" \
-    -cf "${GIT_TARBALL_NAME}" .
+
+if [ "$(printf '%s\n' "$TAR_VERSION" "1.28" | sort -V | head -n1)" == "1.28" ]; then
+    tar --sort=name \
+        --mtime="@${SOURCE_DATE_EPOCH}" \
+        --owner=0 --group=0 --numeric-owner \
+        --xform="s%^\./%${GIT_ARCHIVE_PREFIX}%" \
+        --exclude pkgs --exclude .git* --exclude "${GIT_TARBALL_NAME}" \
```

Comment:
Please quote `.git*`.

Reply:
Oops already done but I forgot to commit this one.